src/DeviceListener.ts

```diff
@@ -298,21 +298,15 @@ export default class DeviceListener {
                 showSetupEncryptionToast(SetupKind.VERIFY_THIS_SESSION);
                 this.checkKeyBackupStatus();
             } else {
-                const backupInfo = await this.getKeyBackupInfo();
```

Comment:
The "There are 3 different toasts for" first part of the multipart comment needs updating

Reply:
Good catch!